Build failure - master branch renamed: awesome-tmux

Since 2026/1/9 the "master" branch of rothgar/awesome-tmux/ has gone 404. Exploration suggests it's now called "main":

[https://api.github.com/repos/rothgar/awesome-tmux/contents/README.md?ref=main](https://api.github.com/repos/rothgar/awesome-tmux/contents/README.md?ref=main)

Log of the failure (including a working access, for comparison):

```
INFO [156/674] Fetching source: sdras/awesome-actions
INFO 156/674 try updating sdras/awesome-actions/README.md
INFO README.md is up to date, cause sha1 is same
INFO [157/674] Fetching source: rothgar/awesome-tmux
INFO 157/674 try updating rothgar/awesome-tmux/README.md
error: Uncaught Error: fetch https://api.github.com/repos/rothgar/awesome-tmux/contents/README.md?ref=master failed with status 404
    throw new Error(`fetch ${url} failed with status ${r.status}`);
          ^
    at got (file:///home/runner/work/trackawesomelist-source/trackawesomelist-source/util.ts:767:11)
    at async gotWithCache (file:///home/runner/work/trackawesomelist-source/trackawesomelist-source/util.ts:897:24)
    at async github.getConent (file:///home/runner/work/trackawesomelist-source/trackawesomelist-source/adapters/github.ts:48:16)
    at async default (file:///home/runner/work/trackawesomelist-source/trackawesomelist-source/fetch-sources.ts:136:25)
    at async main (file:///home/runner/work/trackawesomelist-source/trackawesomelist-source/main.ts:52:5)
make: *** [Makefile:34: prod-build] Error 1
Error: Process completed with exit code 2.
```

--- a/config.yml
+++ b/config.yml
@@ -1052,6 +1052,12 @@ sources:
     files:
       README.md:
         index: true
+  clutcher/awesome_jetbrains_plugins:
+    category: Editors
+    default_branch: main
+    files:
+      README.md:
+        index: true
   sindresorhus/awesome-scifi:
     category: Entertainment
     default_branch: main

